docs/fastdds/use_cases/zero_copy/zero_copy.rst

```diff
@@ -150,6 +150,13 @@ it has some constraints:
 *   Only plain types are supported.
 *   Suitable for |PREALLOCATED_MEMORY_MODE-api| and |PREALLOCATED_WITH_REALLOC_MEMORY_MODE-api|
     memory configurations only.
```

Comment:
If we go for the 'let's reference the datasharing constraints instead of repeating them' approach, this can be removed
```suggestion
```